README.md

```diff
@@ -1,13 +1,62 @@
+# A fork of `kcov` for better covering Zig
+
+This is a rather than naive fork of wonderful `kcov` for better supporting Zig projects.
+
+## It adds the ability to auto recognize Zig's `unreachable` and `@panic` for auto ignoring.
+
+Here is a screenshot of auto ignoring `unreachable`.
+
+![zig-unreachable-example](https://github.com/liyu1981/kcov/blob/master/nocover/zig-unreachable.png?raw=true)
+
+Here is another screenshot of auto ignoring `@panic`.
+
+![zig-panic-example](https://github.com/liyu1981/kcov/blob/master/nocover/zig-panic.png?raw=true)
+
+## It also adds the ability in c/c++ source file of using `/* no-cover */` to mark a line to be ignored.
+
+Here is a screenshot of manual ignoring by placing `/* no-cover */` in c source file.
+
+![c-no-cover example](https://github.com/liyu1981/kcov/blob/master/nocover/c-nocover.png?raw=true)
+
+## Usage
+
+There is no change of original `kcov` usage, it will just work. Please follow the below original Kcov README.
+
+### But how can I get the binary?
+
+The best way is to compile from source. It can be done as follows (you will need `cmake`, `ninja`, `llvm@>16` as I tried)
+
+```
+git clone https://github.com/liyu1981/kcov.git
+cd kcov
+mkdir build
+cd build
+CC="clang" CXX="clang++" cmake -G Ninja ..
+ninja
+```
+
+after building is done. The binary is at `build/src/kcov`. Copy somewhere and use it.
+
+(Or you can download a copy of Apple Silicon version binary from the release section.)
```

Comment:
I think we can make a release shortly after merge of this, and then the osx homebrew is usually updated quite quickly.

I suppose that can be added to the README, or another documentation file. Since not all distributions are up to date, I don't want to in general recommend installing from there.

Reply:
same to above